### ISSUE TITLE
Integration fail to load HA 2022.7 v0.18.1

### ISSUE THREAD

**Issue by grewhit25** [bosch-thermostat/home-assistant-bosch-custom-component#173]
**Describe the bug**
After upgrading to HA 2022.7 and Bosch (EasyControl CT200) Component v0.18.1
The integration still fail to load with the following error.

Setup failed for custom integration bosch: Unable to import component: No module named 'bosch_thermostat_client'

**To Reproduce**

**Expected behavior**

**Screenshots**

**Version**
  - HA version? 2022.7.0
  - HA Bosch component version? v0.18.1
  - Python Bosch HTTP client version?

** Debug SCAN **
** IMPORTANT **
Go to Developer tools in Home Assistant, choose Service tab and choose `bosch.debug_scan`
Download file to your computer and upload it somewhere eg. https://jsonblob.com/

**Additional context**

**Comment by grewhit25** [comment]
I have just upgraded to HA 2022.7.1

The component still fail to load:
```logs
Logger: homeassistant.setup
Source: setup.py:162 
First occurred: 08:13:23 (2 occurrences) 
Last logged: 08:13:47

Setup failed for custom integration bosch: Unable to import component: No module named 'bosch_thermostat_client'
```

I re-downloaded the component and restarted HA - without success.
I'm also unable to delete the integration for re-installation.
'Delete' produce the following HA log output:

<details>
  <summary>HA delete log output</summary>

```logs
2022-07-08 08:47:51 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_protocol.py", line 435, in _handle_request
    resp = await request_handler(request)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_app.py", line 504, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.10/site-packages/aiohttp/web_middlewares.py", line 117, in impl
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 222, in forwarded_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 28, in request_context_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 79, in ban_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 236, in auth_middleware
    return await handler(request)
  File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 136, in handle
    result = await result
  File "/usr/src/homeassistant/homeassistant/components/config/config_entries.py", line 84, in delete
    result = await hass.config_entries.async_remove(entry_id)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 897, in async_remove
    await entry.async_remove(self.hass)
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 528, in async_remove
    component = integration.get_component()
  File "/usr/src/homeassistant/homeassistant/loader.py", line 599, in get_component
    cache[self.domain] = importlib.import_module(self.pkg_path)
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/bosch/__init__.py", line 12, in <module>
    from bosch_thermostat_client.const import (
ModuleNotFoundError: No module named 'bosch_thermostat_client'

```
</details>

**Comment by pszafer** [comment]
I'll check it on Monday (till then I'm on vacation)

**Comment by grewhit25** [comment]
No worries; enjoy your vacation.

**Comment by slovdahl** [comment]
What type of HA installation do you use?

**Comment by slovdahl** [comment]
And FWIW, everything works fine for me on HA 2022.7 using the container installation type. And it also worked fine in a HA development environment shortly before the HA release.

**Comment by grewhit25** [comment]
My installation types is Home Assistant Core,
Using; Artifacthub.io [k8-at-home](https://artifacthub.io/packages/helm/k8s-at-home/home-assistant) Chart
with image repository - "ghcr.io/home-assistant/home-assistant",
Python 3.10.5

I will investigate further at my end.

**Comment by grewhit25** [comment]
Update:
I ‘exec’ed into the container, installed the bosh_thermostat_client which enabled me to delete then reinstall the integration; however, that did not solve the problem.

Rolling back to 2022.6.7 solved the problem, rolling forward from here to 2022.7.0; the problem returned:
```logs
2022-07-13 17:37:17 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration bosch: Unable to import component: No module named 'bosch_thermostat_client' 
```
Incidentally, I have two custom integrations with the same issue:
Bosch Thermostat & Alexa_Media

I am currently running HA 2022.6.7 until I have a better understanding of the problem.
Will probably check with the HACS team in the meantime.

Thanks for a great integration btw and for your help on this one.

**Comment by grewhit25** [comment]
Solved with 2022.7.5
